### PR TITLE
fix: adjust alarms

### LINKED ISSUE #204: Fix alarms to be more relevant
Currently alarms are pinging on events that are not actionable, but do it oftenly, leading to possibility of real problems being missed with assumption "it's this small thing again"

### PR DESCRIPTION
**Pull request by Rakowskiii**
# Description

Adjusting alarms to be more relevant and not trigger all the time, with no action to be taken.

Resolves #204 

## How Has This Been Tested?

Deployed to staging

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

File: terraform/monitoring/panels/ecs/availability.libsonnet [hunk 1]
```diff
@@ -1,8 +1,35 @@
...
+
+  conditions = [
+    alertCondition.new(
+      evaluatorParams = [ 99 ],
+      evaluatorType   = 'lt',
+      operatorType    = 'or',
+      queryRefId      = 'Availability',
+      queryTimeStart  = '5m',
+      reducerType     = 'avg',
+    ),
```
Comment (arein) [inline terraform/monitoring/panels/ecs/availability.libsonnet:29]: doesn't work unfortunately. Grafana doesn't allow to report on `expressions`.

File: terraform/monitoring/panels/proxy/errors_provider.libsonnet [hunk 2]
```diff
@@ -20,12 +20,12 @@ local error_alert(vars) = alert.new(
...
   conditions  = [
     alertCondition.new(
-      evaluatorParams = [ 3000 ],
+      evaluatorParams = [ 5000 ],
       evaluatorType   = 'gt',
       operatorType    = 'or',
       queryRefId      = 'bad_gateway',
       queryTimeStart  = '5m',
-      reducerType     = 'sum',
+      reducerType     = 'max',
```
Comment (arein) [inline terraform/monitoring/panels/proxy/errors_provider.libsonnet:28]: why?

**Review by arein — CHANGES_REQUESTED** [review]

File: terraform/monitoring/panels/proxy/errors_provider.libsonnet (hunk 2 above)
Comment (Rakowskiii) [reply terraform/monitoring/panels/proxy/errors_provider.libsonnet:28]: The idea was to make alarms match the graphs (this evaluates every 1 min,so sometimes it alarms on value that's not graphed, which makes sense but has less clarity, I was wondering if we maybe want to instead change it to max and eventually decrease the value to trigger, but I might have misunderstood the way `max` works`) reverted to sum

File: terraform/monitoring/panels/proxy/errors_non_provider.libsonnet [hunk 3]
```diff
@@ -20,12 +20,12 @@ local error_alert(vars) = alert.new(
...
   conditions  = [
     alertCondition.new(
-      evaluatorParams = [ 10 ],
+      evaluatorParams = [ 15 ],
       evaluatorType   = 'gt',
       operatorType    = 'or',
       queryRefId      = 'non_provider_errors',
       queryTimeStart  = '5m',
-      reducerType     = 'sum',
+      reducerType     = 'max',
```
Comment (arein) [inline terraform/monitoring/panels/proxy/errors_non_provider.libsonnet:28]: `max` takes the `max` value from any host. We want `sum`

File: terraform/monitoring/panels/proxy/errors_provider.libsonnet (hunk 2 above)
Comment (arein) [reply terraform/monitoring/panels/proxy/errors_provider.libsonnet:28]: Forgot to revert?

**Review by arein — CHANGES_REQUESTED** [review]

File: terraform/monitoring/panels/proxy/errors_provider.libsonnet (hunk 2 above)
Comment (Rakowskiii) [reply terraform/monitoring/panels/proxy/errors_provider.libsonnet]: I didn't push 🤦  sorry

**Review by arein — APPROVED** [review]

**Pull request merged**